rspec-mode.el

```diff
@@ -705,6 +710,8 @@ file if it exists, or sensible defaults otherwise."
                                          rspec-docker-cwd file))
       (vagrant (replace-regexp-in-string (regexp-quote (rspec-project-root))
                                          rspec-vagrant-cwd file))
+      (rspec-use-relative-path (replace-regexp-in-string (regexp-quote (rspec-project-root))
```

Comment:
You should probably use `file-relative-name` here.

Reply:
I was thinking about doing that first, but then I was not sure if it was always a file coming to this function or if it sometimes is a pattern or similar. The `file-relative-name` works in all cases I have tested so I gladly change to that. Thanks.